app/azure/services/index.js

```diff
@@ -0,0 +1,15 @@
+import angular from 'angular';
+
+import { AzureService } from './azure.service';
+import { ContainerGroupService } from './container-group.service';
+import { ProviderService } from './provider.service';
+import { ResourceGroupService } from './resource-group.service';
+import { SubscriptionService } from './subscription.service';
+
+export default angular
+  .module('portainer.azure.services', [])
+  .service('AzureService', AzureService)
+  .service('ContainerGroupService', ContainerGroupService)
+  .service('ProviderService', ProviderService)
+  .service('ResourceGroupService', ResourceGroupService)
+  .service('SubscriptionService', SubscriptionService).name;
```

Comment:
Shouldn't we use `factory()` here instead of `service()` ? We are returning objects + factories is recommended by [John Papa' style guide](https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#services)

Applicable to all other modules index files